src/core/MOM_barotropic.F90

```diff
@@ -2300,6 +2313,19 @@ subroutine btstep(U_in, V_in, eta_in, dt, bc_accel_u, bc_accel_v, forces, pbce,
       enddo ; enddo
       call post_data(CS%id_Corv_bt, Corv_bt_sum(isd:ied,JsdB:JedB), CS%diag)
     endif
+    if (CS%id_ubtdt > 0) then
+      do j=js,je ; do I=is-1,ie
+        ubt_dt(I,j) = (ubt_wtd(I,j) - ubt_st(I,j))/dt
```

Comment:
Division is a little more expensive than multiplication, so in other parts of the code, the inverse is often calculated and stored near the beginning of the routine (i.e. `Idt = 1./dt`). This expression would then change to:
```
ubt_dt(I,j) = (ubt_wtd(I,j) - ubt_st(I,j))*Idt
```
Most compilers catch this during optimization, but it doesn't really hurt to add it explicitly.

Reply:
Thanks. I will make this change.